dnf/module/repo_module_dict.py

```diff
@@ -105,15 +105,18 @@ def get_module_dependency_latest(self, name, stream, visited=None):
             repo_module_version = repo_module_stream.latest()
             version_dependencies.add(repo_module_version)
 
-            for requires_name, requires_stream in \
-                    repo_module_version.module_metadata.requires.items():
-                requires_ns = "{}:{}".format(requires_name, requires_stream)
-                if requires_ns in visited:
-                    continue
-                visited.add(requires_ns)
-                version_dependencies.update(self.get_module_dependency_latest(requires_name,
-                                                                              requires_stream,
-                                                                              visited))
+            for requires_name, requires_streams in \
+                    repo_module_version.requires():
+                for requires_stream in requires_streams:
+                    if requires_stream[0] == '-':
```

Comment:
what does this mean? is it an exclude? is it handled somewhere?

Reply:
yes, that's an exclude in modulemd

Reply:
dnf cannot handle module excludes yet, that's why it is skipped, as result rpms of skipped module:stream won't get included